### PR TITLE
Update to tr.json

### PR DESCRIPTION
**Pull request by Alpeel**
About ~180 badges.
Also trying to see if I managed to figure out if I managed to figure out Github!

I couldn't figure out how to "undo" commits, so my "repository(?)" does look a little ugly with all the commits, sorry!

### CODE REVIEW

File: lang/tr.json [hunk 1]
```diff
@@ -1,907 +1,907 @@
...
+      "description": "Bu, ismi Kurabiye olan Chihuahua",
+      "condition": "Sam ol"
     },
     "ynfg_dev": {
-      "name": "Fangame Developer",
-      "description": "Reserved for developers of YNFGs hosted on YNOproject",
-      "condition": "Develop or contribute maps to a YNFG hosted on YNOproject"
+      "name": "Hayran Oyunu Geliştiricisi",
+      "description": "YNOproject'de bulunan Yume Nikki Hayran Oyunlarının geliştiricileri için ayrılmıştır",
+      "condition": "YNOproject'de bulunan Yume Nikki Hayran Oyunlarını geliştir veya harita oluşturarak katkıda bulun",
```
Comment (Mimigris) [inline lang/tr.json:112]: You added a comma at the end of said line (line 112) that shouldn't be here

File: lang/tr.json [hunk 2]
```diff
@@ -1,907 +1,907 @@
...
-      "condition": "Activate the Severed Heads event"
+      "name": "Sana Şapkafa Çıkartılır!",
+      "description": "Üzgünüm, dediklerini kafam almadı!",
+      "condition": "Kopmuş Kafalar hadisesini etkinleştir"
     },
     "scared_masada": {
-      "name": "Moonwalk",
-      "description": "Please put the knife away",
-      "condition": "Make Seccom Masada-sensei walk away with the Kitchen Knife effect"
+      "name": "Michael Jackson",
```
Comment (Mimigris) [inline lang/tr.json:312]: I wonder if this reference may not be too direct, though I suppose it's up to you

File: lang/tr.json [hunk 3]
```diff
@@ -1,907 +1,907 @@
...
-      "name": "Psychedelic Folk",
-      "condition": "Visit the Underground Lake, the Cliffside Woods, and the Radiant Stones Pathway"
+      "name": "Psikedelik Halk",
+      "condition": "Yer Altı Gölü, Uçurum Kenarı Koruluğu ve Işıldayan Taşlar Patikası'nı ziyaret et"
     },
     "singularity": {
-      "name": "Miniature Singularity",
-      "condition": "Find and interact with All 5 Black Hole Girls in Río's worlds. If you have already done so previously, you must visit the area where you found them."
+      "name": "Minyatür Tekillik",
+      "condition": "Río'nun dünyalarındaki 5 Kara Delik Kız'ın tamamını bul ve etkileşime geç. Eğer önceden bunu yaptıysan, onları bulduğun alanı tekrar ziyaret etmen gerekiyor"
```
Comment (Mimigris) [inline lang/tr.json:610]: This line is missing a dot at the end, not sure if it's intended

File: lang/tr.json [hunk 4]
```diff
@@ -1,907 +1,907 @@
...
+      "name": "Alevkafa",
+      "description": "Partinin ateşi asla sönmez, denizin altında bile",
+      "condition": "Sualtı Küp Şehri'nde Denizci Disko'su hadisesini tetikle"
     },
     "peak": {
-      "name": "Respite",
-      "description": "In a forest of shadows and horrors askew, something blue blooms anew.",
-      "condition": "View the Peak Event in the Twisted Thickets"
+      "name": "Soluklanma Molası",
+      "description": "Dehşetlerin ve gölgelerin çarpık ormanında, mavi bir şey filizlenir doğada",
```
Comment (Mimigris) [inline lang/tr.json:715]: This line is also missing a dot, also unsure if it's intended

**Comment by Mimigris** [comment]
Also, translating the names of the world isn't necessary a good idea since the location display and wiki link are in English, so this may confuse players when trying to search for info about specific contents.

**Review by Mimigris — CHANGES_REQUESTED** [review]

**Comment by Alpeel** [comment]
> Also, translating the names of the world isn't necessary a good idea since the location display and wiki link are in English, so this may confuse players when trying to search for info about specific contents.

I see, I believed there would be a way to translate the UI element that links to the wiki to different localizations. Would it be better to scrub the world names back to their original at this point?

**Comment by Mimigris** [comment]
While translating the names could be possible for games that are rather static like Yume Nikki, by creating new locations in the repository ynolocations, Yume 2kki is in constant evolution having new maps added and having to be translated each week, so it's globally not a good idea.

**Comment by Alpeel** [comment]
That makes total sense. I will re-commit with the world names undone. This saves a ton of time since world names are by far the hardest to translate, so that is a plus I guess!

File: lang/tr.json [hunk 5]
```diff
@@ -1,907 +1,912 @@
...
-      "name": "Brilliant Bookworm",
-      "condition": "Visit the Library, the Infinite Library, and the Mystery Library"
+      "name": "Dahiyane Kitap Kurdu",
+      "condition": "Library, Infinite Library ve Mystery Library'yi ziyaret e"
     },
     "cosmic_cube": {
-      "name": "Cosmic Cube",
-      "condition": "Reach Cosmic Cube World"
+      "name": "Kozmik Küp",
+      "condition": Cosmic Cube World'e ulaş"
```
Comment (Mimigris) [inline lang/tr.json:711]: You removed a " at the start of this line

**Review by Mimigris — CHANGES_REQUESTED** [review]

**Comment by Mimigris** [comment]
The changes made in this pull request got added and merged with #749 and are now implemented on the site, so this pull request can safely be closed 👍

**Pull request closed without merging**